Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -572,6 +432,122 @@ public static void WhiteLabelRequestVerification(int userID, Action<LootLockerRe
+        /// 
+        /// Depending on response of this method the developer can either start a session using the token,
+        /// or show a login form.
+        /// 
+        /// White label platform must be enabled in the web console for this to work.
+        /// </summary>
+        public static void CheckWhiteLabelSession(string email, string token, Action<bool> onComplete)
+        {
+            if (!CheckInitialized(true))
+            {
+                onComplete(false);
+                return;
+            }
+
+            VerifyWhiteLabelSession(email, token, onComplete);
+        }
+
+        private static void VerifyWhiteLabelSession(string email, string token, Action<bool> onComplete)
+        {
+            LootLockerWhiteLabelVerifySessionRequest sessionRequest = new LootLockerWhiteLabelVerifySessionRequest();
+            sessionRequest.email = email;
+            sessionRequest.token = token;
+
+            LootLockerAPIManager.WhiteLabelVerifySession(sessionRequest, response =>
+            {
+                onComplete(response.success);
+            });
+        }
+
+        /// <summary>
+        /// Start a game session using the cached white label token and email if any exist
+        /// 
+        /// White label platform must be enabled in the web console for this to work.
+        /// </summary>
+        public static void StartWhiteLabelSession(Action<LootLockerSessionResponse> onComplete)
+        {
+            if (!CheckInitialized(true))
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerSessionResponse>());
+                return;
+            }
+
+            string existingSessionToken = PlayerPrefs.GetString("LootLockerWhiteLabelSessionToken", "");
+            if (string.IsNullOrEmpty(existingSessionToken))
+            {
+                onComplete(LootLockerResponseFactory.Error<LootLockerSessionResponse>("no session token found"));
+                return;
+            }
+
+            string existingSessionEmail = PlayerPrefs.GetString("LootLockerWhiteLabelSessionEmail", "");
+            if (string.IsNullOrEmpty(existingSessionEmail))
+            {
+                onComplete(LootLockerResponseFactory.Error<LootLockerSessionResponse>("no session email found"));
+                return;
+            }
+
+            LootLockerWhiteLabelSessionRequest sessionRequest = new LootLockerWhiteLabelSessionRequest() { email = existingSessionEmail, token = existingSessionToken };
+            LootLockerAPIManager.WhiteLabelSession(sessionRequest, onComplete);
+        }
+
+        /// <summary>
+        /// Start a game label session using the provided email and password
+        /// 
+        /// White label platform must be enabled in the web console for this to work.
+        /// </summary>
+        public static void StartWhiteLabelSession(string email, string password, Action<LootLockerSessionResponse> onComplete)
```

Comment:
password isn't used for this endpoint.
It is like this in the SDK currently, so not your wrongdoing.
But maybe make a new one for this endpoint without password and mark this one as obsolete/deprecated, since this endpoint requires a token to be sent instead.
I'm also a bit confused, does this function actually work?
According to the Game API doceumentation, this should fail; 
https://ref.lootlocker.com/game-api/#white-label-authentication 
or password would be ignored at least, or we support starting a session like this but don't have it documented.

Reply:
It does work, for legacy reasons. But agree that we should split it and mark the old behaviour and deprecated

Reply:
I think I fixed it now.